USAGE.md

```diff
@@ -327,6 +327,25 @@ Potential spelling errors will be printed in the continuous integration log alon
 Words in `build/assets/custom-dictionary.txt` are ignored during spellchecking.
 Spellchecking is currently only supported for English language manuscripts.
 
+## AI-assisted authoring
+
+The workflow [`ai-revision`](.github/workflows/ai-revision.yaml) is available to assist authors in writing their manuscripts.
+It uses large language models (LLM) to revise the manuscript text, fixing spelling and grammar errors, and improving the sentence structure and the writing style with section-specific prompts.
+It is manually triggered by the user (it never runs automatically), and it generates a pull request with suggested revisions.
+Then the user can review these changes and merge the pull request if they are acceptable.
+More information about this tool is available in [this manuscript](https://greenelab.github.io/manubot-gpt-manuscript/).
+
+You must provide a secret with name `OPENAI_API_KEY` containing your OpenAI API token.
+The easiest way to set this is to go to your organization's settings page and, within "Secrets and variables," select "Actions."
+Next, create a secret with the appropriate name and the value of the API token.
+By default, the tool uses the model `text-davinci-003`.
+Make sure to check the [pricing](https://openai.com/api/pricing/) of the OpenAI API.
+The cost for a revision of a standard manuscript should be around $0.50.
```

Comment:
How many words is a "standard" manuscript?

Reply:
The median is 4,133 words, according to the [first Google result asking this](https://quantifyinghealth.com/length-of-a-research-paper/#:~:text=1%2D%20The%20median%20length%20of,between%202%2C023%20and%208%2C284%20words.). That's about 34 paragraphs, so it should be around that ($0.50). Our [manuscript about this tool](https://greenelab.github.io/manubot-gpt-manuscript/) had 34 paragraphs, and it cost $0.34.

I added this info in the sentence.